test/common/testlib.py

```diff
@@ -191,7 +222,10 @@ def focus(self, selector):
         self.call_js_func('ph_focus', selector)
 
     def key_press(self, keys):
-        return self.phantom.keys('keypress', keys)
+        for k in keys:
+            if k == 'Return':
+                k = '\r'  # PhantomJS backwards compat
```

Comment:
This was mostly while developing this PR to avoid having to touch the tests too much. I think we should do this, but as a followup. I added it to the "follow-up cleanup" list in the description.